src/main/java/org/reaktivity/specification/http/internal/Functions.java

```diff
@@ -35,6 +35,20 @@ public static String loginBase64Encoder(String login)
     }
 
     @Function
+    public static byte[] header(String name, String value)
+    {
+        int nameLength = name.length(), valueLength = value.length();
+        int length = nameLength + valueLength + 3;
+        byte[] result = new byte[length];
+        result[0] = (byte) 0x00;
+        result[1] = (byte) nameLength;
+        System.arraycopy(name.getBytes(), 0, result, 2, nameLength);
+        result[2 + nameLength] = (byte) valueLength;
+        System.arraycopy(value.getBytes(), 0, result, 3 + nameLength, valueLength);
+        return result;
```

Comment:
Use `HttpHeaderFW.Builder` generated from `http.idl` to layout bytes onto a `DirectBuffer`, then allocated `byte[]` based on encoded length and copy bytes from `DirectBuffer` to `byte[]` via `DirectBuffer.getBytes()`.

Use `nukleus-maven-plugin` to generate the flyweights and flyweight builders.

Note: no need to check in generated code, similar approach to existing nukleus implementation projects.